ci/install_conda.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 
 if [ -d "$HOME/miniconda3" ] && [ -e "$HOME/miniconda3/bin/conda" ]; then
-    echo "Miniconda install already present from cache: $HOME/miniconda3"
-    rm -rf $HOME/miniconda3/envs/hosts  # Just in case...
+    echo "Miniconda install already present in cache: $HOME/miniconda3"
+    rm -rf "$HOME"/miniconda3/envs/hosts  # Just in case...
```

Comment:
The double quote is not necessary for our use case.

Reply:
Even if it's not necessary I went with consistency in this patch.